content/en/apps/examples/training.md

```diff
@@ -30,16 +30,25 @@ The CHT’s onboarding and training capabilities offer a remote way to provide e
 
 ## User Roles Example
 
-![user-roles.png](user-roles.png)
+![user-roles.png](training/user-roles.png) 
 
 ## Reporting Hierarchy Example
 
-![hierarchy.png](hierarchy.png)
+![hierarchy.png](training/hierarchy.png)
 
-## Workflow Example
+## Workflow Examples
 
-This SMS based workflow example illustrates how the CHT enables remote training, tasking and communication at scale. Training can be done using simple guides, audio/videos suitable for low bandwidth, or interactive experiences on personal devices. Training programs can be easily configured to suit specific health program needs.
+![workflow.png](training/workflow1.png)
```

Comment:
The `training` folder is automatically used since it is the file name. It means that the image won't show when reading the markdown file in GitHub, but it will when processed into the doc site by Hugo.